templates/default.ini

```diff
@@ -66,7 +66,7 @@ qos = string(default="regular")
 nodes = integer(default=1)
 walltime = string(default="02:00:00")
 mapping_file = string(default="")
-vars = string(default="FSNTOA,FLUT,FSNT,FLNT,FSNS,FLNS,SHFLX,QFLX,TAUX,TAUY,PRECC,PRECL,PRECSC,PRECSL,TS,TREFHT,CLDTOT,CLDHGH,CLDMED,CLDLOW")
+vars = string(default="FSNTOA,FLUT,FSNT,FLNT,FSNS,FLNS,SHFLX,QFLX,TAUX,TAUY,PRECC,PRECL,PRECSC,PRECSL,TS,TREFHT,CLDTOT,CLDHGH,CLDMED,CLDLOW,U")
```

Comment:
Minor detail I should have caught earlier: we don't need globally averaged U by default, you can remove it here. But we do need it above for the regridded time series.